frontend/src/scenes/surveys/utils.test.ts

```diff
@@ -287,97 +291,11 @@ describe('survey utils', () => {
                 detractors: 14,
                 passives: 0,
                 promoters: 0,
+                score: '-100.0',
                 total: 14,
             })
         })
     })
-
-    describe('calculateNps', () => {
```

Comment:
those tests are removed because i refactored the `calculateNpsBreakdown` to already return the score on it. no need for a function just to calculate the score